reference/forms/types/collection.rst

```diff
@@ -53,13 +54,9 @@ in a form. For example, suppose you have an ``emails`` field that corresponds
 to an array of email addresses. In the form, you want to expose each email
 address as its own input text box::
 
-    use Symfony\Component\Form\Extension\Core\Type\CollectionType;
-    use Symfony\Component\Form\Extension\Core\Type\EmailType;
-    // ...
-
-    $builder->add('emails', CollectionType::class, array(
+    $builder->add('emails', 'Symfony\Component\Form\Extension\Core\Type\CollectionType', array(
```

Comment:
Why the change away from the `CollectionType::class`? That option (with `::class`) *seems* a little easier because we can use IDE auto-completion. Or have we been changing this elsewhere?